src/main/cli/plots.py

```diff
@@ -31,63 +40,68 @@ def str_to_participants_distribution_statistics_key(cls, value: str) -> STATISTI
             if value.lower() == v.value.lower():
                 return v
         available_values_message = ', '.join(list(map(lambda v: v.value, available_values)))
-        raise argparse.ArgumentTypeError(f'{value} is not a valid type_dstr value. Available values: {available_values_message}')
+        raise argparse.ArgumentTypeError(f'{value} is not a valid {PLOTS_PARAMS.TYPE_DISTR.value} value. '
+                                         f'Available values: {available_values_message}')
 
     @classmethod
     def str_to_chart_type(cls, value: str) -> CHART_TYPE:
         try:
-            return CHART_TYPE.str_to_chart_type(value)
-        except ValueError as e:
-            raise argparse.ArgumentTypeError(str(e))
+            return CHART_TYPE(value.lower())
+        except ValueError:
+            available_values_message = ', '.join(list(map(lambda c_t: c_t.value, CHART_TYPE)))
+            raise argparse.ArgumentTypeError(f'{value} is not a chart_type value. '
+                                             f'Available values: {available_values_message}')
 
     @classmethod
     def str_to_extension(cls, value: str) -> EXTENSION:
         available_extensions = [EXTENSION.HTML, EXTENSION.PNG]
         available_extensions_message = ', '.join(list(map(lambda e: e.value.strip('.'), available_extensions)))
         error_message = f'{value} is not a valid extension. Available values: {available_extensions_message}'
         try:
-            extension = EXTENSION(value.strip('.'))
+            extension = EXTENSION(add_symbol_to_begin(value.rstrip('.'), '.'))
```

Comment:
What's the reason to rstrip a dot and then add it again? Especially when you check for dot presence in  add_symbol_to_begin()